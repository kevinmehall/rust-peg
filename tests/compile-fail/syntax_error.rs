extern crate peg;

peg::parser!(grammar foo() for str {
    fn asdf() {} //~ ERROR expected ":"
});

fn main() {}
