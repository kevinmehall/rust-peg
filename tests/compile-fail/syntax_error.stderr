error: expected ":"
 --> $DIR/syntax_error.rs:4:8
  |
4 |     fn asdf() {} //~ ERROR expected ":"
  |        ^^^^
